app/assets/javascripts/angular/modules/news-title-parse.js

```diff
@@ -0,0 +1,9 @@
+angular.module('parseNews', []).filter('parseTitle', function () {
```

Comment:
Noam wanted the mobile version of the news to be "SCIENCE & FINANCIAL TIMES" instead of "SCIENCE & FINANCIAL TIMES REPORTS:". This is a quick helper to cover that difference between the Dashboard and Mobile versions. It would be better to simply remove "Reports:" from the database and add it into the Desktop view, but I'm not 100% sure how the media names are generated now and frankly this felt easier for mvp.